### PR TITLE
Java Release Notes 3.45.0

### PR DESCRIPTION
**Pull request by MatKuhr**
## What Has Changed?

See title

## Manual Checks?

- [x] Text adheres to the style guide (`vale docs/`)
  - Every sentence is on its own line
  - Headings use [title capitalization](https://capitalizemytitle.com/style/AP/#) (applies also to the sidebar and title of a document)
  - You followed [naming center](https://www.sapbrandtools.com/naming-center/#/dashboard) guidelines when referring to SAP products (e.g. SAP S/4HANA)
- [x] You checked your spelling and grammar (consider using Grammarly, see `CONTRIBUTING.md`)
- [x] You formatted all changed files with prettier (`npm run prettier`)
- [x] You tested if the documentation still builds (`npm run build`)
- [x] You verified all new and changed links still work (changing the `id` or name of a file can break links)
- [x] You have updated the [feature matrix](https://github.com/SAP/cloud-sdk/blob/main/docs/components/data/features.js) if you add documentation on a new feature or otherwise required.

### CODE REVIEW

File: docs/java/release-notes.mdx [hunk 1]
```diff
@@ -54,6 +54,110 @@ To continue using the latest features outlined in the release notes for version
...
+    - Minor version updates:
+      - Update JSON Smart from `2.4.2` to `2.4.7`
+      - Update Togglz from `2.9.6` to `2.9.8`
+      - Update Fabric Gateway Java from `2.2.1` to `2.2.2`
+      - Update jOOQ from `3.14.8` to `3.14.9`
+      - Update Hibernate Core from `5.4.39.Final` to `5.4.40.Final`
+
+### Fixed issues
+
+- Using `null` in OData v2 filter expressions does no longer lead to a `NullPointerException` upon executing the request.
```
Comment (newtork) [inline docs/java/release-notes.mdx:158]:
```suggestion
- Using `null` in OData v2 filter expressions does no longer lead to a `NullPointerException` when executing the request.
```

File: docs/java/release-notes.mdx [hunk 2]
```diff
@@ -54,6 +54,110 @@ To continue using the latest features outlined in the release notes for version
...
+- Fix an edge case where `BigDecimal` values where not serialised correctly in OData V2 requests.
```
Comment (newtork) [inline docs/java/release-notes.mdx:159]:
```suggestion
- Fix an edge case where `BigDecimal` values where not serialized correctly in OData V2 requests.
```

Comment (MatKuhr) [reply docs/java/release-notes.mdx]: > serialization (US spelling) or serialisation (UK spelling)

I always wondered which of the two is correct, seems like both are :D

File: docs/java/release-notes.mdx [hunk 3]
```diff
@@ -54,6 +54,110 @@ To continue using the latest features outlined in the release notes for version
...
+  service.batch()
+    .beginChangeSet().
+    .addFunctionImport(service.callSomeFunction())
+    .endChangeSet()
+    .executeRequest(destination);
+  ```
+
+- For the [Generic OData Client](https://sap.github.io/cloud-sdk/docs/java/features/odata/generic-untyped-odata-client) filterable values and fields can now use the `in` operator in expressions.
+
+  ```
```
Comment (Johannes-Schneider) [inline docs/java/release-notes.mdx:77]:
```suggestion
  ```java
```

**Review by Johannes-Schneider — CHANGES_REQUESTED** [review]

**Review by Johannes-Schneider — APPROVED** [review]

**Pull request merged**